Using real-world clinical data and multi-omics approaches, this study aims to reveal the role of neutrophil extracellular traps in AS. We analyzed clinical records and blood test results from AS patients between January 2012 and February 2024, dividing the data into chronological internal and external datasets.  Machine learning algorithms identified AS-associated risk factors, with SHAP and LIME methods providing interpretability.  Bone marrow samples underwent 10X single-cell RNA sequencing to characterize the AS immune microenvironment.  Mendelian randomization pinpointed critical immune-related genes, while immunofluorescence assays detected NETs in AS specimens. The XGB model demonstrated strong performance, achieving an AUC of 0.760 in internal validation, which outperforming other algorithms.  This predictive capability held in external validation (AUC=0.863).  SHAP and LIME interpretation identified neutrophil as significant AS risk factors.  In single-cell sequencing, Granulocyte sub clustering yielded 16 subsets, with neutrophil populations.  Mendelian randomization highlighted PADI4 as a crucial neutrophil related gene, a finding confirmed by immunohistochemistry showing elevated PADI4 in AS samples.  Importantly, immunofluorescence demonstrated co-localization of NE and citrullinated histone H3, indicating active NETs formation.
